src/Symfony/Component/Console/Tests/Helper/SymfonyQuestionHelperTest.php

```diff
@@ -101,6 +103,15 @@ public function testAskEscapeLabel()
         $this->assertOutputContains('Do you want a \?', $output);
```

Comment:
This test does not longer look relevant, and anyway I think it didn't covered the [initial issue](https://github.com/symfony/symfony/issues/19493) 😕 

Changing it for the following:

```php

    public function testAskEscapeLabel()
    {
        $helper = new SymfonyQuestionHelper();
        $helper->setInputStream($this->getInputStream('sure'));
        $helper->ask($this->createInputInterfaceMock(), $output = $this->createOutputInterface(), new Question('Add a \\'));

        $this->assertOutputContains('Add a \\', $output);
    }
```

will result into the following failure:

```php
 1) Symfony\Component\Console\Tests\Helper\SymfonyQuestionHelperTest::testAskEscapeLabel
Failed asserting that ' Add a </info>:
 > ' contains "Add a \".
```

Looks edgy to have a simple backslash at the end, but it actually was the initial issue. It seems there is no need to escape anything else 😕